test/e2e/storage/csi_mock_volume.go

```diff
@@ -137,7 +137,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 		if dDriver, ok := m.driver.(testsuites.DynamicPVTestDriver); ok {
 			sc = dDriver.GetDynamicProvisionStorageClass(m.config, "")
 		}
-		nodeName := m.config.ClientNodeName
+		nodeName := m.config.ClientNodeSelector[v1.LabelHostname]
```

Comment:
nit: helper function for getting nodename through config. better not duplicate logic for getting a specific key from a map.